galaxy_ng/tests/integration/api/test_pulp_api.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from ansible.galaxy.api import GalaxyError
+
+from ..utils import get_client
+
+
+@pytest.mark.standalone_only
+@pytest.mark.pulp_api
+def test_pulp_api_redirect(ansible_config, artifact):
```

Comment:
This seems out of place with the rest of the PR to me. Useful test, not sure it makes sense in this PR

Reply:
ah